includes/Modules/Analytics.php

```diff
@@ -1070,6 +1075,21 @@ function( $property ) use ( $existing_tag ) {
 		return $response;
 	}
 
+	/**
+	 * Gets the existing tag IDs.
+	 *
+	 * @return array
+	 */
+	private function get_existing_tag() {
```

Comment:
I'm wary of introducing this, as it's only relevant until we finish the work on #340. We actually may wanna consolidate the logic to more closer rely on the actual request parameters (i.e. name these array keys `existing...` instead of without the prefix, which btw can also confuse when reading the code).

Reply:
> We actually may wanna consolidate the logic to more closer rely on the actual request parameters (i.e. name these array keys existing... instead of without the prefix

That would be fine with me I think; this isn't intended to be long lived. We need to store it in the property temporarily either way so I think this is about as explicit as we can be for now, until #340 is done.

Are you suggesting renaming these keys in the returned array here or something else?